app/assets/javascripts/directives/bDropzone.coffee

```diff
@@ -35,23 +35,34 @@
     element.bind 'drop', (event) ->
       if event?
         event.preventDefault()
-      reader = new FileReader()
-      reader.onload = (evt) ->
-        # TODO: add some security checks?
-        #if event.originalEvent.dataTransfer.files[0].type in validMimeTypes
-          # update bindings
-        scope.$apply ->
-          scope.file = evt.target.result
-          scope.object.data = reader.result.split(",")[1]
-          console.log scope.object
-          scope.drop = true
-          if file.type in ['image/jpeg', 'image/png']
-            scope.image = evt.target.result
+      files = event.originalEvent.dataTransfer.files
+      i = 0
+      temp = []
+      while i < files.length
+        addFile = (file) ->
```

Comment:
I read that it has to be closed up within a loop to be able to handla multiple files.